rundeckapp/templates/config/log4j2.properties.template

```diff
@@ -35,6 +35,18 @@ appender.audit.policies.type = Policies
 appender.audit.policies.time.type = TimeBasedTriggeringPolicy
 appender.audit.policies.time.interval = 1
 
+appender.audit_events.type = RollingFile
+appender.audit_events.name = audit_events
+appender.audit_events.fileName = ${baseDir}/rundeck.audit.events.log
+appender.audit_events.append = true
+appender.audit_events.bufferedIO = true
+appender.audit_events.filePattern = ${baseDir}/rundeck.audit.events.log.%d{yyyy-MM-dd}.gz
+appender.audit_events.layout.type = PatternLayout
+appender.audit_events.layout.pattern = ${prefix} - %m%n
+appender.audit_events.policies.type = Policies
+appender.audit_events.policies.time.type = TimeBasedTriggeringPolicy
+appender.audit_events.policies.time.interval = 1
```

Comment:
i think all of the log4j config updates also need to be added to the [packaging repo](https://github.com/rundeck/packaging/blob/main/lib/common/etc/rundeck/log4j2.properties)

Reply:
created PR https://github.com/rundeck/packaging/pull/36